docs/services/gitlab-ci.md

```diff
@@ -403,4 +403,112 @@ Inactivity is defined as, but not limited to:
 * No new commits to an existing project
 * Prolonged period of continuously failing CI/CD jobs (In the case of re-occurring scheduled jobs)
 
----
+## GitLab REST API
+The GitLab REST API provides programmatic access to read and modify GitLab resources by using standard HTTP
+methods and JSON data formats.[^1]
+The examples below demonstrate how to fetch the logs and artifacts from a GitLab pipeline in a GitLab project.
+
+In order to use the REST API,  a personal access token is required.
+Please follow the instructions at
+[GitLab Personal Access Token](https://docs.gitlab.com/user/profile/personal_access_tokens/) documentation to
+create one.
+This token is used to authenticate the user and must be kept secure and not shared with others.
+Next, we need to find the project id for the GitLab project.
+In order to find the project ID, navigate to the project homepage in the GitLab web interface and click the
+button with three vertical dots in the upper right corner of the page and then click `Copy project ID` (see
+the figure below).
+
+<figure markdown>
+  ![GitLab Project ID](files/gitlab-ci/GitLabProjectID.png){ width="700" }
+  <figcaption>GitLab Project ID screenshot</figcaption>
```

Comment:
No need to include HTML tags for including figures

Also replace `<figcaption>` tags with https://facelessuser.github.io/pymdown-extensions/extensions/blocks/plugins/caption/